components/brave_rewards/common/rewards_util.cc

```diff
@@ -40,10 +45,49 @@ bool IsDisabledByFeature() {
   return false;
 }
 
+bool IsOFACSanctionedRegion(const std::string& country_code) {
+  return brave_l10n::IsISOCountryCodeOFACSanctioned(country_code) ||
+         brave_l10n::IsUNM49CodeOFACSanctioned(country_code);
+}
+
+std::string CountryCodeFromCountryId(int country_id) {
+  std::string country_code = "  ";
+  country_code[1] = country_id & 0xFF;
+  country_code[0] = (country_id >> 8) & 0xFF;
+  return country_code;
+}
+
+std::string& MutableCountryCodeStringForTesting() {
+  static base::NoDestructor<std::string> country_code;
+  return *country_code;
+}
+
+const std::string& CountryCodeStringForTesting() {
+  return MutableCountryCodeStringForTesting();
+}
+
+const std::string GetCountryCode() {
+  return CountryCodeStringForTesting().empty()
+             ? brave_l10n::GetDefaultISOCountryCodeString()
+             : CountryCodeStringForTesting();
+}
```

Comment:
Can we not use `brave_l10n::test::ScopedDefaultLocaleForTesting`>

Reply:
Updated in f2087ee34a0e253cdbe78496396c14f9033c1fe6